lib/access_check_filter.js

```diff
@@ -64,6 +66,9 @@ module.exports = (hyper, req, next, options, specInfo) => {
                 }
 
                 return contentPromise.then((content) => redirect(content, location, options));
+            } else if (content.status === 302) {
```

Comment:
@gwicke nono. We're only setting 302 in parsoid.js when the content was *not* in storage because in that case restriction table update would be concurrent with the content request because we've just parsed out the redirect.